NEWS.md

```diff
@@ -1,3 +1,8 @@
+## 24.0.2 2022-08-10
+
+* Fixing when migration script to populate holdings effective locations runs ([MODINVSTOR-940] (https://issues.folio.org/browse/MODINVSTOR-940))
+* There is an operator script (on the wiki in inventory->scripts) that can be used to manually populate effectiveLocationId fields in holdings records for libraries who still have difficulties getting this field populated for existing records
```

Comment:
I would not include any operator scripts in the release notes. By definition, they are not part of the module itself.

Reply:
Ah, of course.  When she said release notes, I assumed she meant the ones that go out with a release version-this makes much more sense.  I'll amend, merge, and release.